### ISSUE TITLE
dbConnect fails in version 1.4.0 with oracle

### ISSUE THREAD

**Issue by apalacio9502** [r-dbi/odbc#700]
Hello,

I was testing the ODBC 1.4.0 version and I found that when making the connection it gave me the following error.

```
Error in initialize(value, ...) :
   invalid names for slots of class “Oracle”: ptr, quote, info, encoding
```

I leave an example of how I am making the connection, this example works in version 1.3.4 and 1.3.5.

Passing the arguments
```
  DBI::dbConnect(
    odbc::odbc(),
    Driver = "Oracle",
    Host = "----------",
    SVC = "----------",
    UID = "----------",
    PWD = "----------",
    Port = 1521,
    timezone = "America/Bogota",
    timezone_out = "America/Bogota"
  )
```

Using the odbc.ini configuration

```
[database-test]
Driver = oracle
Host = ----------
Port = 1521
SVC = ----------
UID = ----------
PWD = ----------

```
```
  DBI::dbConnect(
    odbc::odbc(),
    dsn= "database-test",
    timezone = "America/Bogota",
    timezone_out = "America/Bogota"
  )
```

Regards,

**Comment by detule** [comment]
Thanks for the report.  We noticed this as well and fixed the problem, though the fix has not yet been released to CRAN.

In the meantime you can either install the development version or follow some of the suggestions in https://github.com/r-dbi/odbc/issues/685 

Thanks again

**Comment by apalacio9502** [comment]
Thank you for your prompt response, do you have any plans to update the version in CRAN. We were waiting for the new version that corrected the errors of version 1.3.5 "Fix regression when falling back to odbcConnectionColumns() to describe column data types".

**Comment by hadley** [comment]
I'll probably submit an update to CRAN today or tomorrow.

Closing this as a duplicate of #685
